test/e2e/install/install_test.go

```diff
@@ -399,6 +396,58 @@ func overallDeployment() []client.Object {
 	return deployedObj
 }
 
+func deleteObject(obj client.Object) {
+	err := e2eclient.Client.Delete(context.TODO(), obj)
+	if err != nil {
+		Expect(errors.IsNotFound(err)).To(BeTrue(), fmt.Sprintf("unexpected error: %v", err))
```

Comment:
Assertion outside of the ginkgo spec?

Reply:
the `deleteObject` helper didn't make too much sense in the first place. Removed.